src/locale/locales/fr/messages.po

```diff
@@ -4044,52 +3790,43 @@ msgstr "Voir les posts <0>{displayTag}</0> de ce compte"
 #: src/view/com/notifications/FeedItem.tsx:419
 #: src/view/com/util/UserAvatar.tsx:381
 msgid "See profile"
-msgstr ""
+msgstr "Voir le profil"
 
 #: src/view/screens/SavedFeeds.tsx:164
 msgid "See this guide"
 msgstr "Voir ce guide"
 
-#: src/view/com/auth/HomeLoggedOutCTA.tsx:40
-#~ msgid "See what's next"
-#~ msgstr "Voir la suite"
-
 #: src/view/com/util/Selector.tsx:106
 msgid "Select {item}"
 msgstr "Sélectionner {item}"
 
 #: src/screens/Login/ChooseAccountForm.tsx:61
 msgid "Select account"
-msgstr ""
+msgstr "Sélectionner un compte"
 
 #: src/screens/Login/index.tsx:120
 msgid "Select from an existing account"
 msgstr "Sélectionner un compte existant"
 
 #: src/view/screens/LanguageSettings.tsx:299
 msgid "Select languages"
-msgstr ""
+msgstr "Sélectionner les langues"
 
 #: src/components/ReportDialog/SelectLabelerView.tsx:30
 msgid "Select moderator"
-msgstr ""
+msgstr "Sélectionner le modérateur"
 
 #: src/view/com/util/Selector.tsx:107
 msgid "Select option {i} of {numItems}"
 msgstr "Sélectionne l’option {i} sur {numItems}"
 
-#: src/view/com/auth/create/Step1.tsx:96
-#: src/view/com/auth/login/LoginForm.tsx:153
-#~ msgid "Select service"
-#~ msgstr "Sélectionner un service"
-
 #: src/screens/Onboarding/StepSuggestedAccounts/index.tsx:52
 msgid "Select some accounts below to follow"
 msgstr "Sélectionnez quelques comptes à suivre ci-dessous"
 
 #: src/components/ReportDialog/SubmitView.tsx:133
 msgid "Select the moderation service(s) to report to"
-msgstr ""
+msgstr "Sélectionnez le(s) service(s) de modération auquel(s) vous devez vous adresser"
```

Comment:
Oh, there is a mistranslation here (for once!), as it would mean `Select the moderation service(s) you need to talk with`, which is not really the meaning of the original sentence.

I had to rework the sentence to make it more natural in French, meaning approximately `Select the moderation service(s) that will be the recipients of the report".

```suggestion
msgstr "Sélectionnez le(s) service(s) de modération destinataires du signalement"
```